ci/coverity.sh

```diff
@@ -16,6 +16,14 @@ CC=cc
 
 cov-configure --config ${COV_CONFIG} --compiler ${CC} --comptype gcc --template
 cov-build --dir ${COV_DIR} --config ${COV_CONFIG} make
+
+# gcc >= 10.0 workaround, see https://www.mail-archive.com/gcc@gcc.gnu.org/msg93814.html
+if grep -qr 'version>0' . --include=*.xml ; then
```

Comment:
shouldn't we check for gcc version before doing this? Also, can you share the xml file path.

Reply:
The `if` checks if the version that coverity found starts with `0`, which is the bug when the version is actually `10`.

The xml files are in `${COV_DIR}/emit`, `coverity-intermediate/emit/` under the build directory with our settings in the script.

Reply:
instead of directly searching the version in xml file, why don't we do this if the gcc version if greater than or equal to 10?

Reply:
The `if grep -qr 'version>0' . --include=*.xml` checks for the actual bug. So if coverity fixes the bug we wouldn't do the workaround either. So this is more reliable than just testing for the gcc version.